### ISSUE TITLE
`/drip` doesn't seem to handle `duration` correctly

### ISSUE THREAD

**Issue by AlexV525** [sharat87/httpbun#16]
It's more like seconds/10 rather than seconds.

![image](https://github.com/sharat87/httpbun/assets/15884415/24f570a8-fe55-4522-a03c-d21bb56e01ee)

**Comment by sharat87** [comment]
Looks to be working fine for me? 🤔

![shot-2023-12-03-07-27-37](https://github.com/sharat87/httpbun/assets/120119/0d27419b-68c5-48de-aec3-ca0d3543e0fc)

**Comment by AlexV525** [comment]
Which means you implemented it as a total duration rather than the interval?
![image](https://github.com/sharat87/httpbun/assets/15884415/3798ce34-bc11-4a6d-9ac2-a0dabb3b2fda)

**Comment by sharat87** [comment]
Indeed. I intended for `duration` to be the total duration and implemented it that way. I can see how that line in the docs can be confusing. Let me see if I can rephrase.

I checked on httpbin.org, it documents as
![shot-2023-12-03-07-56-37](https://github.com/sharat87/httpbun/assets/120119/ce671878-4dd4-48a5-9624-b99d06df7b2b)

And the implementation is that it's the total duration, just like what we have in httpbun. :/

**Comment by AlexV525** [comment]
Great! At least it resolves my confusion

**Comment by sharat87** [comment]
Thanks. Just pushed a change in the docs and a better error message on bad request.
